samoa-api/src/main/java/org/apache/samoa/evaluation/EvaluatorProcessor.java

```diff
@@ -179,6 +211,28 @@ private void addMeasurement() {
     }
   }
 
+  /**
+   * This method is used to create one line of a text file containing predictions and votes (for classification only).
+   * In case, this is the first line a header line is also added
+   */
+  private void addVote() {
+    List<Vote> votes = new Vector<>();
+    Collections.addAll(votes, evaluator.getPredictionVotes());
+    Vote[] finalVotes = votes.toArray(new Vote[votes.size()]);
```

Comment:
I think there is no need to build an array to hold the votes, you can pass the collection right through.